templates/server-acl-init-job.yaml

```diff
@@ -45,7 +45,8 @@ spec:
             - "-ec"
             - |
               consul-k8s server-acl-init \
-                -release-name={{ .Release.Name }} \
+                -server-label-selector=component=server,app={{ template "consul.name" . }},release={{ .Release.Name }} \
```

Comment:
@ishustava your idea of using the label selector already fixed a bug! Previously we were assuming the `app` label was always `consul` however it can actually be overridden by the `nameOverride` config value! 🍻

Reply:
Nice!